src/Files.App/Views/Properties/DetailsPage.xaml

```diff
@@ -91,14 +98,18 @@
 				<ListView.Footer>
 					<Button
 						x:Name="ClearPropertiesButton"
-						Margin="0,4,0,0"
+						Margin="0,8,0,0"
 						HorizontalAlignment="Stretch"
-						Content="{helpers:ResourceString Name=ClearPropertiesButton/Content}">
+						Content="{helpers:ResourceString Name=ClearPropertiesButton/Content}"
+						Style="{StaticResource EnhancedButtonStyle}">
```

Comment:
What does this style look like?

Reply:
EnhancedButtonStyle is same as SettingsButtonStyle. Properties pages use buttons styled SettingsButtonStyle so I thought it should be styled with the same one as well

Reply:
We want to limit the use of the custom style for the navigation menu. Regular buttons should use the regular style.

Reply:
I believe 'Clear All Properties' is treated like as a Card button(fully expanded to the horizontal).

Reply:
I agree, but in that case we should use the settings control (like we do in the about page in settings).

Reply:
Reverted